core/crypto.mk

```diff
@@ -203,7 +203,11 @@ _CFG_CORE_LTC_CE := $(CFG_CRYPTO_WITH_CE)
 _CFG_CORE_LTC_VFP := $(CFG_WITH_VFP)
 _CFG_CORE_LTC_BIGNUM_MAX_BITS := $(CFG_CORE_BIGNUM_MAX_BITS)
 _CFG_CORE_LTC_PAGER := $(CFG_WITH_PAGER)
-_CFG_CORE_LTC_OPTEE_THREAD := $(CFG_LTC_OPTEE_THREAD)
+ifneq ($(CFG_NUM_THREADS),1)
+$(call force,_CFG_CORE_LTC_OPTEE_THREAD,y)
+else
+$(call force,_CFG_CORE_LTC_OPTEE_THREAD,n)
+endif
```

Comment:
That works but since `_CFG` variables are supposed to be internal and not set from the environment / command line, using `force` is not really appropriate. I suggest `:=` like before.

Reply:
ok, fair.